.gitignore

```diff
@@ -37,3 +37,16 @@ _sketches.cpp
 
 # local override files
 Makefile.local
+
+# downloaded package sources
+pkg/cmsis-dsp/cmsis-dsp
+pkg/libcoap/libcoap
+pkg/libfixmath/libfixmath
+pkg/microcoap/microcoap
+pkg/micro-ecc/micro-ecc
+pkg/oonf_api/oonf_api
+pkg/openwsn/openwsn
+pkg/relic/relic
+pkg/tinydtls/tinydtls
+pkg/tlsf/tlsf
+pkg/wakaama/wakaama
```

Comment:
I would prefer to add these to a new `.gitignore` in the package's directory. See [my lwip package](https://github.com/authmillenon/RIOT/blob/pkg/port/lwip/pkg/lwip/.gitignore) for an example.